common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisControlManager.java

```diff
@@ -112,7 +112,7 @@ public void tick(Level level) {
                 this.onFlightEnd();
             }
 
-            if (level.getGameTime() % (20 * 1.75) == 0 && ticksLanding == 0) {
```

Comment:
Return this line, but make it so it plays until 2 seconds left before the "drum" sound.